app/javascript/components/pages/PostNew.js

```diff
@@ -54,26 +59,19 @@ class ApartmentNew extends Component {
 	}
 
 	render() {
-		console.log(this.state.pic)
+		// console.log("picture:",this.state.picture)
+		console.log("pic inside of form:",this.state.form.pic)
+		console.log("form:",this.state.form)
+		console.log("state:", this.state)
 		return (
 			<React.Fragment>
-				{/* <h3>Add an Post</h3>
+				<h3>Add a Post</h3>
 				<div className='body-container'>
 					<div className='form'>
+					<img src={this.state.form.pic} height='600' width='600' alt='' />
```

Comment:
does the payload come with a description of image? might be good to use it for alt tag for accessibility